pages/about/groups/agwg.md

```diff
@@ -33,67 +34,80 @@ Some information on this page is also shown on, and may be more current in, the
+To become a participant requires a commitment from you to support the work of the group in the following ways:
 
-### Meeting Minutes
+* Engage actively with other participants in the working group using the group’s communication channels.
+* Keep up with weekly tasks and the progress of the work &mdash; for example, via the minutes of past meetings, mailing list discussions, and GitHub issue comments.
+* Give your input on time.
 
-[Minutes from previous meetings](https://www.w3.org/WAI/GL/minutes-history) are available.
+To learn how you can become a participant in the group, see [Instructions for joining the Accessibility Guidelines Working Group](https://www.w3.org/groups/wg/ag/instructions/).
 
-## Administrative Information
+## Group members and task forces
 
-Work of the AG WG is in accordance with the [W3C Process](https://www.w3.org/2023/Process-20231103/). AG WG work is funded in part by the [WAI Core 2015 Project](https://www.w3.org/WAI/about/projects/wai-core-2015/). The work of this group does not necessarily reflect the views or policies of the funders.
+* [Chairs](https://www.w3.org/groups/wg/ag/participants/#chairs)
+* [Current participants](https://www.w3.org/groups/wg/ag/participants/#participants)
+* [Task forces](https://www.w3.org/groups/wg/ag/task-forces/) that support the AG Working Group
 
-The chairs of the AG WG, responsible for overall leadership and management, are Chuck Adams, Alastair Campbell, and Rachael Montgomery. The staff contact, responsible for W3C Process and general support, is [Kevin White](https://www.w3.org/People/kevin/). Administrative inquiries may be sent to [group-ag-chairs@w3.org](mailto:group-ag-chairs@w3.org).
+## Communications and decisions
 
-Work on WCAG 3 is additionally led by Shawn Lauriat and Jeanne Spellman, task force co-facilitators, and Wilco Fiers, WCAG 3 Project Manager. Inquiries about WCAG 3 should be sent to [group-ag-plan@w3.org](mailto:group-ag-plan@w3.org) to reach these people as well as the chairs.
+* [Minutes from previous meetings](https://www.w3.org/WAI/GL/minutes-history)
+* [Mailing list archive](https://lists.w3.org/Archives/Public/w3c-wai-gl/)
+* [Record of decisions](https://www.w3.org/WAI/GL/wiki/Decisions)
+* [Decision policy](/about/groups/agwg/decision-policy/)
 
-The AG WG maintains the following operational resources:
+## Publications and copyright
 
-* [Decision policy](decision-policy);
-* [Record of decisions made by the WG](https://www.w3.org/WAI/GL/wiki/Decisions);
-* [meeting minutes](https://www.w3.org/WAI/GL/minutes-history).
+* [List of publications by the AG Working Group](https://www.w3.org/groups/wg/ag/publications/)
+* [Intellectual property rights &mdash; AG Working Group](https://www.w3.org/groups/wg/ag/ipr/)
 
-W3C maintains a [public list of any patent disclosures](https://www.w3.org/groups/wg/ag/ipr/) made in connection with the deliverables of the group; that page also includes instructions for disclosing a patent.
+## Contact the chairs
```

Comment:
This might benefit from being slightly higher up the page. Possibly before 'Contribute to the work'

Reply:
If you think the contact information is relevant to the whole page, my preference is to leave the contact details last in the page navigation. 
I've noticed in user testing when people are looking for contact details on an unfamiliar web page, they usually expect to find this information at the bottom. 
The conventional pattern sees action links always follow learning links (where people first figure out whether the information is useful to them before taking action). 
It's the same learn-then-act pattern as a subscribe link or a follow (on social media) link. 
If you think the information is only relevant to a particular section on the page,  we could make the heading an h3 and place it under 'Contribute to the work' (after 'Become a participant in the group')?
I feel it's more useful to visitors at the end, though.